src/EnergyPlus/DataEnvironment.cc

```diff
@@ -181,6 +181,122 @@ namespace DataEnvironment {
 
 	// Functions
 
+	// Clears the global data in DataEnvironment.
+	// Needed for unit tests, should not be normally called.
+	void
+	clear_state()
```

Comment:
You seem to be setting up a scenario where namespaces could become classes and perhaps their ctor take idf text or alternatively a data structure produced by your parser.  ??